checkstyle/checkstyle.xml

```diff
@@ -292,14 +291,20 @@
             <property name="max" value="15"/>
         </module>
         <module name="ClassFanOutComplexity">
-            <property name="max" value="40"/>
+            <!-- TODO figure out why this needed an increase -->
+            <property name="max" value="42"/>
         </module>
         <module name="CyclomaticComplexity">
             <property name="severity" value="error"/>
             <property name="max" value="12"/>
         </module>
         <module name="NPathComplexity">
-            <property name="max" value="50"/>
+            <!-- TODO Agree on the value
+                 Checkstyle 7.7 improved the NPath algorithm further that leads to higher NPath counts.
+                 7.6 passes with max 50, 7.7 needs it to be set to at least 216
+                 the default threshold is 200
+            -->
+            <property name="max" value="250"/>
```

Comment:
See an explanatory calculation in `HazelcastManifestTransformer` below, which is the method with greatest NPath count 216 in OSS.